sources/packages/web/src/components/common/OverawardDetails.vue

```diff
@@ -0,0 +1,225 @@
+<template>
+  <v-card class="mb-5">
+    <v-container>
+      <body-header
+        title="Overawards"
+        subTitle="Overaward amounts generated due to application reassessments"
+      />
+      <content-group>
+        <toggle-content :toggled="!overawards.length">
+          <DataTable
+            :value="overawards"
+            class="p-m-4"
+            :paginator="true"
+            :rows="pageLimit"
+            :rowsPerPageOptions="PAGINATION_LIST"
+          >
+            <Column field="dateAdded" header="Date added">
+              <template #body="slotProps">
+                <span>
+                  {{ dateOnlyLongString(slotProps.data.dateAdded) }}
+                </span>
+              </template>
+            </Column>
+            <Column field="applicationNumber" header="Application #"
+              ><template #body="slotProps">
+                <span>
+                  {{ slotProps.data.applicationNumber ?? "-" }}
+                </span>
+              </template></Column
+            >
+            <Column field="overawardOrigin" header="Origin"></Column>
+            <Column field="assessmentTriggerType" header="Type"></Column>
+            <Column field="awardValueCode" header="Award"></Column>
+            <Column field="overawardValue" header="Overaward amount">
+              <template #body="slotProps">
+                <span>
+                  {{ formatCurrency(slotProps.data.overawardValue, "") }}
```

Comment:
We can remove the `currencyCode = "(CAD)"` from the `formatCurrency` as per the last conversations with @JasonCTang and @hellolynn-tbtb, and then the last empty string parameter will not be required here.